src/pages/Admin/Sidebar/Sidebar.styles.ts

```diff
@@ -2,10 +2,8 @@ import styled from "styled-components";
 
 export const Wrapper = styled.div`
   display: flex;
-  height: 100vh;
```

Comment:
Why don't we need to set the height on this anymore? Presumably we want it to be set to 100% or similar?